### PR TITLE
Camera api

### PR DESCRIPTION
**Pull request by tomermoshe**

### CODE REVIEW

File: src/services/camera/camera.service.ts [hunk 1]
```diff
@@ -0,0 +1,219 @@
+import { Injectable } from '@angular/core';
+import { CesiumService } from '../cesium/cesium.service';
+import { SceneMode } from '../../models/scene-mode.enum';
+
+/**
+ *  The service exposes the scene's camera and screenSpaceCameraController
```
Comment (eitanfr) [inline src/services/camera/camera.service.ts]: add a small doc about the scene mode and how it affects tilt , rotate...

File: src/components/ac-map/ac-map.component.ts [hunk 2]
```diff
@@ -61,26 +64,26 @@ export class AcMapComponent implements OnChanges, OnInit {
...
-   * @type {boolean}
-   */
-  @Input()
-  enableTilt: boolean = AcMapComponent.DEFAULT_TILT_ENABLE;
+  // /**
+  //  * in meters
+  //  * @type {number}
+  //  */
+  // @Input()
+  // minimumZoom: number = AcMapComponent.DEFAULT_MINIMUM_ZOOM;
```
Comment (eitanfr) [inline src/components/ac-map/ac-map.component.ts]: not sure if to keep zoom props

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,16 +1,19 @@
 ## 0.0.16
```
[the remark this answers is not in the document]
Comment (tomermoshe) [reply CHANGELOG.md]: done

File: src/models/scene-mode.enum.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/**
+ * Cesium scene modes
+ */
+export enum SceneMode {
```
[the remark this answers is not in the document]
Comment (tomermoshe) [reply src/models/scene-mode.enum.ts]: it is exported in models.ts

File: src/services/camera/camera.service.ts (hunk 1 above)
Comment (tomermoshe) [reply src/services/camera/camera.service.ts]: done

**Pull request merged**